catalog/content/docs/src/main/resources/ExtContents.adoc

```diff
@@ -1,18 +1,22 @@
 
-== Overview
+This section supports developers creating extensions of the existing framework. 
+There are currently no {branding} Content development details beyond those covered in the {branding} Developer's Guide.
 
-The DDF Content application provides a framework for storing, reading, processing, transforming and cataloging data.
+=== Whitelist
 
-This guide supports developers creating extensions of the existing framework. There are currently no DDF Content development details beyond those covered in the DDF Developer's Guide.
+The following packages have been exported by the {branding} Content Application and are approved for use by third parties:
 
-== Whitelist
-
-The following packages have been exported by the DDF Content Application and are approved for use by third parties:
-
-* ddf.content
-* ddf.content.data
-* ddf.content.operation
-* ddf.content.plugin
-* ddf.content.storage
-* ddf.content.util
-* ddf.content.core.directorymonitor
\ No newline at end of file
+`ddf.camel.component.content`
```

Comment:
Just checking... is it truly your intent to have these all render inline with just one blank space separating each one? Are all package lists moving from bullet lists over to this style?